LayoutTests/imported/w3c/web-platform-tests/css/css-shapes/shape-functions/circle-function.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Shapes Module Level 1: parsing the circle() function</title>
+<link rel="help" href="https://drafts.csswg.org/css-shapes-1/#funcdef-basic-shape-circle">
+<meta name="assert" content="Tests parsing of the circle() function">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/parsing-testcommon.js"></script>
+<style>
+    #target {
+        width: 200px;
+        height: 100px;
+    }
+</style>
+</head>
+<body>
+<div id="target"></div>
+<script>
+    test_valid_value("shape-outside", "circle()");
+    test_valid_value("shape-outside", "circle(1px)");
+    test_valid_value("shape-outside", "circle(20px at center)", "circle(20px)");
+    test_valid_value("shape-outside", "circle(at 10% 20%)");
+    test_valid_value("shape-outside", "circle(4% at top right)", "circle(4% at 100% 0%)");
+    test_valid_value("shape-outside", "circle(calc(100% - 20px) at calc(100% - 20px) calc(100% / 4))", "circle(calc(100% - 20px) at calc(100% - 20px) calc(25%))");
+
+    test_valid_value("shape-outside", "circle(closest-corner at center)", "circle(closest-corner");
+    test_valid_value("shape-outside", "circle(closest-corner at 20px 50px)", "circle(closest-corner at 20px 50px");
+    test_valid_value("shape-outside", "circle(closest-side at center)", "circle()");
+    test_valid_value("shape-outside", "circle(closest-side at 20px 30%)", "circle(at 20px 30%)");
+    test_valid_value("shape-outside", "circle(farthest-corner at center top)", "circle(farthest-corner at 50% 0%)");
+    test_valid_value("shape-outside", "circle(farthest-corner at center)", "circle(farthest-corner)");
+    test_valid_value("shape-outside", "circle(farthest-side at center top)", "circle(farthest-side at 50% 0%)");
+    test_valid_value("shape-outside", "circle(farthest-side at center)", "circle(farthest-side)");
+
+    test_invalid_value("shape-outside", "circle(123)");
+    test_invalid_value("shape-outside", "circle(at)");
+    test_invalid_value("shape-outside", "circle(4% 20%)");
+    test_invalid_value("shape-outside", "circle(4% 20% at center)");
+    test_invalid_value("shape-outside", "circle(4px 20px)");
+    test_invalid_value("shape-outside", "circle(4px, 20px)");
+    test_invalid_value("shape-outside", "circle(at 4px, 20px)");
+    test_invalid_value("shape-outside", "circle(-10px at 20px 30px)");
+    test_invalid_value("shape-outside", "circle(-10% at 20% 30%)");
+    test_invalid_value("shape-outside", "circle(1% 2% at 0% 100%)");
```

Comment:
Usually we have `-valid.html` and `-invalid.html` (and also `-computed.html`) all as separate files. Can you make that consistent?